addons/mail/static/src/messaging/component/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -842,6 +842,134 @@ QUnit.test('open 2 different chat windows: enough screen width', async function
     );
 });
 
+QUnit.only('open 2 chat windows: check right shift operations are available', async function (assert) {
+    assert.expect(9);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                channel_type: "channel",
+                id: 20,
+                name: "General",
+            }],
+            channel_direct_message: [{
+                channel_type: "chat",
+                direct_partner: [{
+                    id: 7,
+                    name: "Demo",
+                }],
+                id: 10,
+            }],
+        },
+    });
+    await this.start({
+        debug: true,
+        async mockRPC(route, args) {
+            if (args.method === 'channel_fetch_preview') {
+                return [{
+                    id: 20,
+                    last_message: {
+                        author_id: [7, "Demo"],
+                        body: "<p>test</p>",
+                        channel_ids: [20],
+                        id: 100,
+                        message_type: 'comment',
+                        model: 'mail.channel',
+                        res_id: 20,
+                    },
+                }, {
+                    id: 10,
+                    last_message: {
+                        author_id: [7, "Demo"],
+                        body: "<p>test2</p>",
+                        channel_ids: [10],
+                        id: 101,
+                        message_type: 'comment',
+                        model: 'mail.channel',
+                        res_id: 10,
+                    },
+                }];
+            }
```

Comment:
couldn't populate `this.data` and let default mocked `channel_fetch_preview` do it for you?

Reply:
just c/c'ed previous tests, I'll check that